static/css/components/header-bar.less

```
@@ -91,13 +91,13 @@
   }
 
   .app-drawer {
-    @topHeight: 50px;
+    @topHeight: 0px;
     position: fixed;
     width: 300px;
     max-width: 100vw;
     right: 0;
     top: @topHeight;
-    height: calc(100% - @topHeight - 25px);
+    height: 100%;
```

```suggestion
    height: 100vh;
```
Lets use 100vh as it makes it clear that this is indented to be the full viewheight of the screen.